packages/headless/doc-parser/src/entity-builder.ts

```diff
@@ -64,8 +64,23 @@ export function buildParamEntity(param: Parameter): Entity {
   };
 }
 
+export function buildReturnTypeEntity(m: ApiMethodSignature): Entity {
+  const desc = getReturnTypeDescription(m);
+
+  return {
+    kind: 'primitive',
+    name: 'returnType',
+    type: m.returnTypeExcerpt.text,
```

Comment:
Do we need to run the `type` through `sanitizeType` to remove $1? If it's not needed, then all the better,

Reply:
Yep, it works fine without it as it stands now. If we run into a case in the future that needs it, we can add it then 👍🏻